libs/state/README.md

```diff
@@ -2,28 +2,25 @@
 
 [![rx-angular circleci-status](https://circleci.com/gh/BioPhoton/rx-angular.svg?style=shield)](https://circleci.com/gh/BioPhoton/rx-angular)
 
-#### Reactive Component State for Angular
+## Reactive Component State for Angular
 
-RxState is a light-weight reactive state management service especially useful for component state in Angular.
-Furthermore a global service is provided and can act as a small global state manager.
+RxState is a light-weight, flexible, strongly typed and tested tool dedicated to reduce the complexity of managing component state in angular.
 
 ![state logo](https://raw.githubusercontent.com/BioPhoton/rx-angular/master/libs/state/images/state_logo.png)
 
 ## Description
 
-In most of the component-oriented applications, there is the need to structure container components.
-Even after a well thought refactoring into more display components and grouping logic into responsibilities it's always hard to handle.
+Developing **reactive**, **complex** and **robust** user interfaces confronts developers with manging an overwhelmingly huge amount
+events of different types and sources _aka_ **State Management**.
```

Comment:
The sentence is confuded IMHO

Reply:
will split up the sentence in mangaging/transforming state and handling/bundling events